FRPs/proposal-26-security.md

```diff
@@ -0,0 +1,32 @@
+---
+id: 26
+title: Quantifying the Impact of MEV on blockchain consensus security
+team: Liyi Zhou, Kaihua Qin, Arthur Gervais
+created: 2020-12-10
+paper: 2
+---
+
+# Summary
+Multiple works have expressed the qualitative intuition that MEV deteriorates (PoW) blockchain consensus security. Yet, we’re not aware of any quantitative results supporting this hypothesis. The literature proposes various methods on quantifying (PoW) blockchain security. One common insight is that the stale block rate aggravates the risks of double spending and selfish mining. That means, the more forks we can observe on-chain, the easier game becomes for an adversary to defraud other blockchain participants. If MEV encourages miners to fork the blockchain, MEV would directly deteriorate the blockchain’s consensus security. In this FRP we aim to quantitatively reason about the impact of MEV on blockchain security.
+
+## Background and Problem Statement
+In this FRP, we would like to find the minimum MEV value, at which an MEV-aware miner will start to fork the blockchain to claim MEV. If this threshold is very low (e.g., < 10 times the block reward), then MEV clearly is a danger to the chain. Further, we want to extend the study to a multiplayer game, where multiple MEV-aware miners would engage to compete over an MEV opportunity. We then aim to quantify the average and maximum fork length that we can expect to observe.
+
+By parameterizing a blockchain through its stale block rate, we can capture any PoW blockchain reparametrization (e.g., covering Bitcoin, Ethereum and others). While the results are expected to primarily apply to PoW based blockchains, we would like to investigate to what degree our model would also apply to PoS.
+
+## Plan and Deliverables
+- Within this work, we aim to provide a model that allows us to identify the optimal mining strategies to exploit MEV given one MEV-aware miner. Given the optimal mining strategies we can then determine the minimum MEV amount, at which a rational MEV-miner would attempt to fork the blockchain. Our results are aimed to be applicable to any PoW blockchain following the longest chain rule.
```

Comment:
Another cool thing to include would be a model / equation which allows calculating the number of confs to consider a transaction safe, under some amount of MEV + hashrate willing to go after it

Reply:
In this vein I'd also love to see an "MEV risk" dashboard, which just estimates overall MEV in the network in real-time using heuristics.  Could even be a client plugin to give users e.g. an extra warning when MEV is high.  A set of heuristics for "MEV danger" would be another cool outcome of analyzing this style of game.

Reply:
Ideally we should be able to provide a transaction value, above which an MEV hashrate will start to go after it. Not sure if the number of confirmations matter then. Our model should hopefully provide this threshold MEV value to quantify the MEV risk.